waspc/data/Generator/templates/react-app/package.json

```diff
@@ -6,8 +6,9 @@
   {=& depsChunk =},
   {=& devDepsChunk =},
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
+    "validate-env": "node ./scripts/validate-env.mjs",
+    "start": "npm run validate-env && react-scripts start",
+    "build": "npm run validate-env && react-scripts build",
```

Comment:
Why can't the server fail like this too?

Reply:
I wanted to keep things inside of the `src` folder so I don't have to use `mjs`, but maybe in the name of consistency it's better to do same things in Web App and Server.

Reply:
I've updated the server to work exactly like the web app.